Source/View Controller/WorkshopTableViewController.swift

```diff
@@ -75,8 +75,7 @@ internal class WorkshopTableViewController: NavigationBarTableViewController, Na
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         
-        guard
-            let cell = tableView.dequeueReusableCell(withIdentifier: WorkshopTableViewCell.identifier, for: indexPath) as? WorkshopTableViewCell,
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: WorkshopTableViewCell.identifier, for: indexPath) as? WorkshopTableViewCell,
             indexPath.section < viewModel.viewContent.count,
```

Comment:
Multi-guard statements should be on new lines to make it easier to read. Makes it easier to see what ends where

Reply:
I did the suggested edits although I'm not sure that this is what you wanted. Did you mean something different?